packages/snap/src/rpc/utxo/bitcoin/handlers.ts

```diff
@@ -27,6 +28,19 @@ export const bitcoinGetAddress = async (
   }
 }
 
+export const bitcoinGetPublicKeys = async (
+  params: BitcoinGetAddressParams,
+): Promise<PublicKey[]> => {
+  try {
+    const bitcoinSigner = new BitcoinSigner()
+    await bitcoinSigner.initialize()
+    return await bitcoinSigner.getPublicKeys(params)
+  } catch (error) {
+    moduleLogger.error({ fn: 'bitcoinGetPublicKeys' }, error)
+    return Promise.reject(error)
+  }
+}
```

Comment:
That was the whole magic we needed to be able to derive pubkeys (as in, xpubs) at a specific path - no need to fiddle manually deriving from the root xpriv, the signer is an hdwallet instance, so use it